genesis/Cargo.toml

```diff
@@ -15,5 +15,6 @@ iroha_schema = { version = "=2.0.0-pre-rc.13", path = "../schema" }
 derive_more = { version = "0.99.17", default-features = false, features = ["deref"]}
 serde = { version = "1.0.151", features = ["derive"] }
 serde_json = "1.0.91"
+json5 = "0.4.1"
```

Comment:
The whole point behind introducing JSON5 was to make use of comments and other syntactic sugar. I'd suggest either hard forking out genesis block and expanding it with explanatory comments (more work for `kagami` less work for us, when we're trying to explain stuff to users), or making `genesis.json` be strict JSON.

Reply:
Ok, let's for now make `genesis.json` strict JSON, we can later expand it to JSON5.